src/domain/search/filters/filter/DateFilter.js

```diff
@@ -5,21 +5,25 @@ import DayPicker from 'react-day-picker';
 import MomentLocaleUtils from 'react-day-picker/moment';
 import ControlLabel from 'react-bootstrap/lib/ControlLabel';
 import FormGroup from 'react-bootstrap/lib/FormGroup';
-import Glyphicon from 'react-bootstrap/lib/Glyphicon';
 import InputGroup from 'react-bootstrap/lib/InputGroup';
 import Overlay from 'react-bootstrap/lib/Overlay';
+import FormControl from 'react-bootstrap/lib/FormControl';
+import Button from 'react-bootstrap/lib/Button';
 
+import constants from '../../../../../app/constants/AppConstants';
 import injectT from '../../../../../app/i18n/injectT';
 import iconCalendar from './images/calendar.svg';
 
-const DatePickerWrapper = ({ children }) => (
-  <div className="app-DateFilter__datePicker">
+const DatePickerWrapper = ({ children, ...rest }) => (
+  <div aria-hidden={rest['aria-hidden']} className="app-DateFilter__datePicker" tab-index={rest['tab-index']}>
```

Comment:
In React `tab-index` should be `tabIndex`.

Reply:
Good catch 👍While fixing this I noticed that there's unexpected behaviour when tabbing into this component, I'll fix that as well.